torch/autograd/_functions/__init__.py

```diff
@@ -1 +1 @@
-from .tensor import *  # noqa: F403
+from .tensor import Type, Resize
```

Comment:
I don't think it is ok to do that. This has been here for a long time and we would need to check usage of this file before doing something like that.